Library/Formula/trafficserver.rb

```diff
@@ -4,27 +4,52 @@ class Trafficserver < Formula
   mirror "https://archive.apache.org/dist/trafficserver/trafficserver-5.2.0.tar.bz2"
   sha1 "a32ae5a0ea6d734a955fa56f3e446a66c08be21f"
 
+  head "https://github.com/apache/trafficserver.git"
+
   bottle do
     sha256 "60ca4def066dd106a5f71706b3b07d7ee8d093f95926d341f4d98bd88c3b4d2d" => :yosemite
     sha256 "25134b656b86365d4c7d43de3038f71e864dfb50c6895f94b3c3b035c3f85986" => :mavericks
     sha256 "4920a708ef78452027f9a258f50b7802b3f0bdc68aca92337d2468d9abe88415" => :mountain_lion
   end
 
+  option "with-spdy", "Build with SPDY protocol support"
+  option "with-brewed-curl", "Build with libcurl from Homebrew"
+
   depends_on "openssl"
   depends_on "pcre"
+  if build.with? :spdy
+    depends_on "spdylay"
+    depends_on "pkg-config" => :build
+  end
+  depends_on "curl" if build.with? "brewed-curl"
+  if build.head?
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool"  => :build
+  end
 
-  # remove some amd64 compiler options that fail on Snow Leopard
-  patch :DATA if MacOS.version == :snow_leopard
+  # patch openssl 1.0.2 tls1.h detection, remove on 5.3.0 (upstream bug TS-3443)
+  patch :DATA unless build.head?
```

Comment:
` if build.stable? ` will work fine here, given there's no devel block to account for.